simpleviewinc/keg-components
# Postinstall update

**Ticket**: n/a
> NOTE: do not merge until re-theme has been updated to the next version. I will then update the package.json to use re-theme v1.1.2. which will fix the gh test action check

## Context

* when importing this repo as an npm package, it will try to run the `postinstall` script. but it will fail because it doesn't exist
* we need to include `/scripts` in the `files` key
* also update the postisntall script to only run when it's not inside `node_modules'

## Goal

* postinstall script wont run if its installed as a node module, but should run if its cloned as a repo

## Updates

* `package.json`
    * removed `scripts` from being included
* added PR template
* added a check to not run postinstall if current dir is in `node_modules`

## Testing
* pull this branch
* run `yarn postinstall`
* verify that it runs the script
     * you should see msg 
```
-----git update assume-unchange folder: build-----
```

* delete the `/scripts` folder entirely
* run `yarn postinstall`
* verify that it does't throw error
